contracts/contracts/PreConfCommitmentStore.sol

```diff
@@ -558,13 +540,13 @@ contract PreConfCommitmentStore is OwnableUpgradeable, UUPSUpgradeable {
         uint256 minTime = block.timestamp - commitmentDispatchWindow;
 
         // Check if the dispatch timestamp is within the allowed dispatch window
-        require(dispatchTimestamp > minTime, "Invalid dispatch timestamp");
+        require(dispatchTimestamp >= minTime, "Invalid dispatch timestamp");
```

Comment:
Is this change of behavior intentional? After clarifying with the auditors their stance is `When inside the require statements, non-strict inequalities (>=, <=) are usually costlier than strict equalities (>, <).`

Reply:
I did it, after reading this quote:
```
When inside the “if/require” statements, non-strict inequalities (>=, <=) are
usually cheaper than the strict equalities (>, <).
```
Thank you for clarifying, I will return it back